### ISSUE TITLE
R-Ladies Lausanne Collaboration (as example)

### ISSUE THREAD

**Issue by sinarueeger** [rladies/rladies-collaborations#2]
Hi @rladies/lausanne!

You have recently **collaborated** with _some company_.

On behalf of R-Ladies global we are keeping track of all the R-Ladies collaborations. 

It is **important** to build a **portfolio** of collaborations that R-Ladies has with others, which can then be used when introducing R-Ladies, but also to keep a record of how R-Ladies is working with its social environment. 

[This repository](https://github.com/rladies/rladies-collaborations) does exactly this. 

➡️ We would love 💜 to have your collaboration documented ⬅️ 

To **submit a new contribution**, please follow these steps:

1. Fork [this repository](https://github.com/rladies/rladies-collaborations) to your own GitHub account.
1. Copy the [template](https://github.com/rladies/rladies-collaborations/blob/master/resources/template.Rmd) and name it `ChapterCity-Organization.Rmd`. Use your chapter name (`ChapterCity`) and the organization's name on the title (`Organization`).
1. Locate the file in the corresponding folder. This should be at: `contributions/year`, where `year` is the year in which the collaboration took part (you might need to create a folder).
    - If several R-Ladies chapters contributed to the same event, you can: (a) put the name of a geographic region, or (b) list their names if they are not more than three chapters (e.g. `ChapterCity1-ChapterCity2-ChapterCity3-Organization.Rmd`).
    - If your collaboration/event started in one year, and ended on the next one, please upload _in the starting year folder_.   
1. Once your file is ready, make a Pull Request (PR) for the RMarkdown file. Available curators will review your submission, and make any suggestion if neccesary.
    - If there are revisions to make, you will have to update the PR, until approved.
    - If you are working on Linux, you may need to install some system packages to get the RMarkdown to work properly. [Check this help](https://dzone.com/articles/installing-latex-ubuntu).
1. Once the review is approved, _knit_ your RMarkdown file to create an `*.html` file, and make a new PR for the HTML file. This will be instantly approved. 

You find **more details** [here](https://github.com/rladies/rladies-collaborations/blob/master/README.md#goals).

If you have never made a **pull request** or have any **question**, please respond in the box below and tag Sina (`@sinarueeger`) or Melina (`@melvidoni`)

**Comment by sinarueeger** [comment]
PR #3 done.
